android-components/components/feature/contextmenu/src/main/java/mozilla/components/feature/contextmenu/ContextMenuCandidate.kt

```diff
@@ -274,7 +280,7 @@ data class ContextMenuCandidate(
             },
             action = { _, hitResult ->
                 val email = hitResult.getLink().stripMailToProtocol()
-                clipPlaintText(
+                clipPlainText(
```

Comment:
nit: Code refactoring / cleanup could be left for separate tickets / commits.

Reply:
Will bear this in mind for future issues 👍